std/zig/parse.zig

```diff
@@ -2689,6 +2689,18 @@ fn SimpleBinOpParseFn(comptime token: Token.Id, comptime op: Node.InfixOp.Op) No
 
 // Helper parsers not included in the grammar
 
+fn parseAmpersandAmpersandOp(arena: *Allocator, it: *TokenIterator, tree: *Tree) !?*Node {
```

Comment:
I'm not too thrilled hardcoding AmpersandAmpersand => Keyword_and, but abstracting a one-off feels overengineered.